### PR TITLE
Popup Window on both desktop and mobile

### PR DESCRIPTION
**Pull request by katarighe**
In this PR, I and my partner worked on it effortlessly adding the popup window and its respective features. 

On the portfolio page, when the user clicks the button to check the project, the popup window together with the details appears in the screen. 

When the user clicks the X button on the popup window, the window disappears completely. 

**For Desktop**
image

**For Mobile**
image

### CODE REVIEW

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ ⚡ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._
 - [ ] The popup window is supposed to fit properly on the screen without scrolling. Your pops need t be  scrolled to fully be seen. Please reduce their sizes so that they fir properly on the screen. 

 **Your desgin on desktop**
toolarge

**Your design on mobile** 
toolarger-mobile

 - [ ] Please style the close button and the line text below the heading as in the Figma. The goal of the project is to reproduce the FIgma
 
 **Your design**
 
top

**Figma**
top1




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -739,3 +835,101 @@ input::placeholder {
...
+
+.modal-body {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 1.25rem;
+  width: 80%;
+  border-radius: 16px;
+  border: 1px solid #ebecf0;
+  box-shadow: 0.1px 48px 48px rgba(37, 47, 137, 0.08);
+}
```
Comment (AbelG101) [inline style.css:861]: - [ ] As mentioned previously please make sure the popup on the mobile version doesn't create an overflow as you can see from the below video it's not fitting proportionally to a screen of size `375px` which is the ideal mobile screen width so please carefully adjust this.
- Your design:
  firefox_p7i54SsgUa
- Figma design:
  image

File: style.css [hunk 2]
```diff
@@ -329,6 +330,18 @@ button:active {
   gap: 6px;
 }
 
+.works_card_history ul {
+  display: flex;
+  gap: 12px;
+}
```
Comment (AbelG101) [inline style.css:336]: - [ ] Good job adjusting the layout of the mini description in the popup but there is still some room for adjustment as I noticed you're forgetting to add the proper color as well as the bullet points between each text, so please make sure to update this in order to have a good replication of the original design.
- Figma design:
  image
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @katarighe 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Popup works as expected ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @katarighe 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Popup works as expected ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @katarighe 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Descriptive PR
✅ Linters passing
## Optional suggestions
- [ ] There is one issue in your hero section. the social icons' pics are breaking. kindly consider fixing them to make your website look professional.
 
Screenshot 2023-06-21 at 8 35 40 PM


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**